server/bot/commands/utilities.ts

```diff
@@ -518,3 +518,56 @@ export const handleEditLastSay: Command = {
     } catch (e) { return 'Nothing to edit.' }
   }
 }
+
+export const handleSuppress: Command = {
+  name: 'suppress',
+  opts: {
+    requirements: {
+      permissions: { manageMessages: true }
+    },
+    description: 'Suppress or unsuppress embeds in a message.',
+    fullDescription: 'Suppress embeds in a message. If the message already has a suppressed embed, it will get unsuppressed. Requires \'Manage Messages\' permission',
+    usage: '/suppress (channel) <message ID or link>',
+    example: '/suppress #general 123456789012345678'
+  },
+  generator: async (message, args, { client }) => {
+    if (args.length === 1) {
+      let msg
+      let channelToEdit
+      const regex = /https?:\/\/((canary|ptb|www).)?discord(app)?.com\/channels\/\d{17,18}\/\d{17,18}\/\d{17,18}/
+      if (regex.test(args[0])) {
+        const split = args[0].split('/')
+        const channel = message.member.guild.channels.get(split[5]) as GuildTextableChannel
+        msg = channel.messages.get(split[6]) || await client.getMessage(split[5], split[6])
+        channelToEdit = split[5]
+      } else {
+        msg = message.channel.messages.get(args[0]) || await message.channel.getMessage(args[0])
+        channelToEdit = message.channel.id
+      }
+      if (msg) {
+        await (client as unknown as { requestHandler: { request: (method: string, url: string, auth: boolean, body: object) => Promise<Object> } }).requestHandler.request('PATCH', `/channels/${channelToEdit}/messages/${msg.id}`, true, { flags: (msg as unknown as { flags: number }).flags ^ Eris.Constants.MessageFlags.SUPPRESS_EMBEDS })
+        message.addReaction('✅')
+      } else return `That's not a real message, you ${getInsult()}`
+    } else if (args.length === 2) {
+      const channelToEdit = getIdFromMention(args[0])
```

Comment:
Use the common variable.
```suggestion
      channelToEdit = getIdFromMention(args[0])
```